mcs/class/System.Web.Services/System.Web.Services/WebMethodAttribute.cs

```diff
@@ -39,7 +39,10 @@ public enum TransactionOption {Disabled , NotSupported , Required , RequiresNew
 #endif
 
 	[AttributeUsage(AttributeTargets.Method, Inherited = true)]
-	public sealed class WebMethodAttribute : Attribute {
+#if !XAMMAC_4_5
```

Comment:
And we can't rip it out, because LogicalMethodInfo.cs/TypeStubManager.cs depends on it, and a bunch of stuff depends on LogicalMethodInfo.cs.